monitoring_service.rst

```diff
@@ -274,3 +167,88 @@ Fields
 +-----------------------+------------+--------------------------------------------------------------------------------------------+
 | signature             | bytes      | Elliptic Curve 256k1 signature on the above data from participant paying the reward        |
 +-----------------------+------------+--------------------------------------------------------------------------------------------+
+
+Appendix A: Interfaces
+======================
+
+Broadcast Interface
+-------------------
+
+Client's request to store a balance proof will be broadcasted using Matrix as a
+transport layer. A public room will be available for anyone to join - clients
+will post balance proofs to the chatroom and Monitoring Services picks them up.
+
+Web3 Interface
+--------------
+
+Monitoring Service are required to have a synced Ethereum node with an enabled JSON-RPC interface. All blockchain
+operations are performed using this connection.
+
+Event Filtering
+'''''''''''''''
+
+MS must filter events for each on-chain channel that corresponds to the submitted balance proofs.
+On ``ChannelClosed`` and ``NonClosingBalanceProofUpdated`` events state the channel was closed with the Monitoring
+Service must call ``updateNonClosingBalanceProof`` with the respective latest balance proof provided by its client.
```

Comment:
Maybe they should call `MonitoringService.monitor()`.  If they directly call `TokenNetwork.updateNonClosingBalanceProof()` directly, they cannot get the reward anymore.

Reply:
Updates this above and removed this whole paragraph.